File: wrappers/mvc/demos/Kendo.Mvc.Examples/Models/SocialBenefits.cs
namespace Kendo.Mvc.Examples.Models
{
    public class SocialBenefits
    {
        public SocialBenefits()
        {
        }

        public SocialBenefits(SocialBenefitType type, double value)
        {
            Type = type;
            Value = value;
        }

        public SocialBenefitType Type { get; set; }
        public double Value { get; set; }
    }
}

File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/models/SocialBenefits.java
package com.kendoui.spring.models;

public class SocialBenefits {
    private SocialBenefitType type;
    private double value;
    
    public SocialBenefits(SocialBenefitType type, double value) {
        this.type = type;
        this.value = value;
    }
    
    public SocialBenefitType getType() {
        return type;
    }

    public void setType(SocialBenefitType type) {
        this.type = type;
    }

    public double getValue() {
        return value;
    }
    
    public void setValue(double value) {
        this.value = value;
    }
}
